dora/shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -308,91 +286,17 @@ private int getProgress(AlluxioURI path, JobProgressReportFormat format,
-        oldLoad(newPath, local);
-      }
-    } else {
-      if (local) {
-        if (!mFsContext.hasNodeLocalWorker()) {
-          System.out.println(
-              "When local option is specified, there must be a local worker available");
-          return;
-        }
-      } else if (status.getInAlluxioPercentage() == 100) {
-        // The file has already been fully loaded into Alluxio.
-        System.out.println(filePath + " already in Alluxio fully");
-        return;
-      }
-      runLoadTask(filePath, status, local);
-    }
-    System.out.println(filePath + " loaded");
-  }
-
-  private void runLoadTask(AlluxioURI filePath, URIStatus status, boolean local)
-      throws IOException {
-    AlluxioConfiguration conf = mFsContext.getPathConf(filePath);
-    OpenFilePOptions options = FileSystemOptionsUtils.openFileDefaults(conf);
-    BlockLocationPolicy policy = Preconditions.checkNotNull(
-        BlockLocationPolicy.Factory
-            .create(conf.getClass(PropertyKey.USER_UFS_BLOCK_READ_LOCATION_POLICY), conf),
-        "UFS read location policy Required when loading files");
-    WorkerNetAddress dataSource;
-    List<Long> blockIds = status.getBlockIds();
-    for (long blockId : blockIds) {
-      if (local) {
-        dataSource = mFsContext.getNodeLocalWorker();
-      } else { // send request to data source
-        BlockStoreClient blockStore = BlockStoreClient.create(mFsContext);
-        Pair<WorkerNetAddress, BlockInStream.BlockInStreamSource> dataSourceAndType = blockStore
-            .getDataSourceAndType(status.getBlockInfo(blockId), status, policy, ImmutableMap.of());
-        dataSource = dataSourceAndType.getFirst();
-      }
-      Protocol.OpenUfsBlockOptions openUfsBlockOptions =
-          new InStreamOptions(status, options, conf, mFsContext).getOpenUfsBlockOptions(blockId);
-      BlockInfo info = status.getBlockInfo(blockId);
-      long blockLength = info.getLength();
-      String host = dataSource.getHost();
-      // issues#11172: If the worker is in a container, use the container hostname
-      // to establish the connection.
-      if (!dataSource.getContainerHost().equals("")) {
-        host = dataSource.getContainerHost();
-      }
-      CacheRequest request = CacheRequest.newBuilder().setBlockId(blockId).setLength(blockLength)
-          .setOpenUfsBlockOptions(openUfsBlockOptions).setSourceHost(host)
-          .setSourcePort(dataSource.getDataPort()).build();
-      try (CloseableResource<BlockWorkerClient> blockWorker =
-          mFsContext.acquireBlockWorkerClient(dataSource)) {
-        blockWorker.get().cache(request);
-      } catch (Exception e) {
-        throw new RuntimeException(String.format("Failed to complete cache request from %s for "
-            + "block %d of file %s: %s", dataSource, blockId, status.getPath(), e), e);
-      }
-    }
+    // TODO(jiacheng): refactor LoadCommand so the main logic is executed here
```

Comment:
In order NOT to incur extra review on the new logic, I will do the bare minimal to make this command just work in this PR. I will follow-up with another PR to resolve this TODO shortly.

Reply:
@jja725 FYI